robot/impl/local_robot.go

```diff
@@ -735,112 +735,67 @@ func (r *localRobot) updateWeakDependents(ctx context.Context) {
 	}
 }
 
-// Config returns the info of each individual part that makes up the frame system
+// config returns the info of each individual part that makes up the frame system
 // The output of this function is to be sent over GRPC to the client, so the client
 // can build its frame system. requests the remote components from the remote's frame system service.
-func (r *localRobot) FrameSystemConfig(ctx context.Context) (*framesystem.Config, error) {
-	localParts, err := r.getLocalFrameSystemParts()
-	if err != nil {
-		return nil, err
-	}
-	remoteParts, err := r.getRemoteFrameSystemParts(ctx)
+func (r *localRobot) FrameSystemConfig(ctx context.Context) ([]*referenceframe.FrameSystemPart, error) {
```

Comment:
Can we rename this and the associated inject function something like `FrameSystemParts`?